manifests/webhook.pp

```diff
@@ -61,11 +61,31 @@
     before => File['webhook_init_script'],
   }
 
+  if $service_file =~ /systemd/ {
+
+    $_service_file_mode = '0644'
+
+    exec { 'webhook_systemd_reload':
```

Comment:
yeah, the implementation may be the same. But I would like to avoid declaring a resource twice, even if it has different names and it would work.